README.md

```diff
@@ -36,3 +36,12 @@ cp configurations/default/env.yml.tmp configurations/default/env.yml
 mvn package
 java -jar target/otp-middleware.jar configurations/default/env.yml
 ```
+
+## OTP Server Proxy Setup
+The follow parameters are used to interact with an OTP server.
+
+| Parameter | Description | Example |
+| --- | --- | --- |
+| OTP_SERVER | This is the address of the OTP server which all OTP related requests will be sent to. | http://otp-server.example.com | 
+| OTP_ROOT_ENDPOINT | This will form the endpoint used by Spark which all OTP requests will be filtered through. | /otp_root_endpoint/ |
+| OTP_PLAN_ENDPOINT | This defines the plan endpoint part of the requesting URL. If a request is made to this, the assumption is that a plan request has been made and that the response should be processed accordingly. | plan_endpoint |
```

Comment:
I think it would be more helpful to provide example values that actually correspond to the path structure of the OTP API. E.g.: 
- OTP_ROOT_ENDPOINT: `/otp/routers/default`
- OTP_PLAN_ENDPOINT: `plan`

EDIT: Oh, after looking at files farther down, I don't actually think OTP_ROOT_ENDPOINT should be a config variable. I think we want that to be stable/constant in the code. As for OTP_PLAN_ENDPOINT, we may just want to check that the request ends with `/plan` instead of being so precise.

Reply:
My first thought on this is that the UI sends the path to the router, and can vary between OTP instances, however I agree with Landon to try to add another configuration point for that.

Reply:
Updated to hardcode the OTP root endpoint and remove the config value.